src/encoded/static/components/publication.js

```diff
@@ -7,7 +7,51 @@ import { DbxrefList } from './dbxref';
 import { PickerActions } from './search';
 import Status from './status';
 import { DisplayAsJson } from './objectutils';
+import { SortTablePanel, SortTable } from './sorttable';
+import { FetchedData, Param } from './fetched';
+
+
+const datasetsColumns = {
+    accession: {
+        display: dataset => <a href={dataset['@id']}>{dataset.accession}</a>,
+        title: 'Accession',
+    },
+    type: {
+        getValue: datasets => (datasets['@type'] || [''])[0], // only the first matters
+        title: 'Type',
+    },
+    target: {
+        getValue: dataset => (dataset.target ? dataset.target.label : ''),
```

Comment:
You didn’t have to change this if you preferred your way, but OK. My personal preferences aren’t gospel, just to be clear. I just found it unusual.